test/e2e/specs/editor/blocks/cover.spec.js

```diff
@@ -297,6 +297,55 @@ test.describe( 'Cover', () => {
 
 		expect( isClickable ).toBe( false );
 	} );
+
+	test( 'can use focal point picker to set the focal point of the cover image', async ( {
+		editor,
+		coverBlockUtils,
+		page,
+	} ) => {
+		await editor.insertBlock( { name: 'core/cover' } );
+		const coverBlock = editor.canvas.getByRole( 'document', {
+			name: 'Block: Cover',
+		} );
+
+		await coverBlockUtils.upload(
+			coverBlock.getByTestId( 'form-file-upload-input' )
+		);
+
+		await editor.selectBlocks( coverBlock );
+
+		const focalPointLeft = page.getByRole( 'spinbutton', {
+			name: 'Focal point left position',
+		} );
+
+		const focalPointTop = page.getByRole( 'spinbutton', {
+			name: 'Focal point top position',
+		} );
+
+		await focalPointLeft.fill( '20' );
+		await focalPointTop.fill( '20' );
+
+		await expect( focalPointLeft ).toHaveValue( '20' );
+		await expect( focalPointTop ).toHaveValue( '20' );
+
+		const blockAttributes = await page.evaluate( () => {
+			const selectedBlock = window.wp.data
+				.select( 'core/block-editor' )
+				.getSelectedBlock();
+			return selectedBlock?.attributes;
+		} );
+
+		expect( blockAttributes.focalPoint ).toMatchObject( {
+			x: 0.2,
+			y: 0.2,
+		} );
```

Comment:
```suggestion
		await expect.poll( editor.getBlocks ).toMatchObject( [
			{
				name: 'core/cover',
				attributes: {
					focalPoint: { x: 0.2, y: 0.3 },
				},
			},
		] );
```

No custom code required, the above is the basic approach to testing for correctness of block attributes.